app/javascript/gobierto_data/webapp/components/sets/SQLEditorCode.vue

```diff
@@ -1,61 +1,62 @@
 <template>
   <div>
     <div class="codemirror">
-      <codemirror
+      <textarea
         ref="myCm"
```

Comment:
"myCm", myCodemirror? use a proper name. What are you gonna type inside the textarea? A query? Then use names like `ref="queryEditor"`